extensions/wikia/PortableInfobox/templates/PortableInfoboxItemHeroMobileExperimental.mustache

```diff
@@ -8,13 +8,11 @@
 		</hgroup>
 	{{/title}}
 	{{#image}}
-		<figure class="article-media" data-ref="{{ref}}">
```

Comment:
can we remove extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage::getGalleryData (if ArticleAsJSON part)
 after SEO friendly images are enabled globally and wikiamobile is removed? If yes could you mark it with appropriate comment